lib/shared/messages.js

```diff
@@ -156,8 +156,8 @@ module.exports = {
 
     notEnoughSpaceInDrive: () => {
       return [
-        'Not enough space on the drive.',
-        'Please insert larger one and try again.'
+        'Not enough space on the target.',
+        'Please insert a larger target and try again.'
```

Comment:
"insert a larger target" sounds weird to me – maybe we should use "insert a larger device" here, or rather "connect a larger device"?